packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -404,52 +407,22 @@ module.exports = {
         minifyURLs: true,
       },
     }),
+    // Makes some environment variables available in index.html.
+    // The public URL is available as %PUBLIC_URL% in index.html, e.g.:
+    // <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    // In production, it will be an empty string unless you specify "homepage"
+    // in `package.json`, in which case it will be the pathname of that URL.
+    new InterpolateHtmlPlugin(env.raw),
```

Comment:
ditto: why reordered?

Reply:
From the top of my head (i think i documented this somewhere) html-plugin switched to sync behaviour in the new plugin API which means it had to be instantiated first to make their own hooks available for tapping - i will double check this. thanks.